swagger_specs.py

```diff
@@ -45,13 +45,13 @@
  - Anonymous functions: require no authentication.
  - Authenticated functions: require asymmetric key authentication. Not tested
    in debug mode.
-    - The 'Pubkey' header will contain the user's pubkey.
-    - The 'Fingerprint' header is constructed from the comma separated
+    - The **'Pubkey'** header will contain the user's pubkey.
+    - The **'Fingerprint'** header is constructed from the comma separated
```

Comment:
@yar-resh , we need to change this into something saner, and I think we need this whole authentication bit to appear in the README of the webserver repo and copied from there to here.